samples/AppConfiguration/AppConfiguration.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.11.0" />
+    <PackageReference Include="Azure.Core" Version="1.12.0" />
```

Comment:
1.13.0 is out

Reply:
Not sure we can go to 1.13 yet since the azure.resourcemanager.core is pointed at 1.12 and then we will get a downgrade error